src/Jira/Api.php

```diff
@@ -156,13 +177,14 @@ public function setEndpoint($url)
 	 */
 	public function getFields()
 	{
-		if ( !count($this->fields) ) {
+		// Fetch fields when the method is called for the first time.
+		if ( $this->fields === null ) {
```

Comment:
I saw this approach in Symfony, but wouldn't `isset($this->fields)` be more explicit (I've wrote about this in another comment)?